KDOP_tree/include/CGAL/KDOP_tree/KDOP_tree.h

```diff
@@ -241,7 +241,7 @@ namespace KDOP_tree {
     /// which `do_intersect` predicates are
     /// defined in the traits class `KDOPTraits`.
     template<typename Query>
-    bool do_intersect(const Query& query) const;
+    bool do_intersect(const Query& query);
```

Comment:
Why do you have to remove constness?  This is only a test.

Reply:
It didn't compile with 'const' last night... I just read the code again and fixed this problem.